src/validators/format.js

```diff
@@ -0,0 +1,20 @@
+const checker = require("../checker");
+const err = require("../errorCodes");
+
+function format(value, expression) {
+  let results = null
+  if (checker.isEmpty(value)) {
+    return null;
+  }
+
+  const result = expression.test(value);
```

Comment:
Since (1) all validators are using checker to check their valuers and (2) in the future we'd like to give a opportunity to inject a different checker, you can move this check to checker as "isValidFormat" or something like this